jdaviz/core/template_mixin.py

```diff
@@ -4691,12 +4691,20 @@ def float_precision(column, item):
                 return ''
             elif isinstance(item, tuple) and np.all([np.isnan(i) for i in item]):
                 return ''
-
             elif isinstance(item, float):
                 return float_precision(column, item)
             elif isinstance(item, (list, tuple)):
                 return [float_precision(column, i) if isinstance(i, float) else i for i in item]
-
+            elif isinstance(item, (np.float32, np.float64)):
+                return float(item)
+            elif isinstance(item, u.Quantity):
```

Comment:
This way we lost the unit info, is that important?

Also I wonder if we can take advantage of the JsonCsutomEncoder that astropy already has for some of these: https://github.com/astropy/astropy/blob/93b6f551b9ed00c4a422b754eae0261f1a7552c9/astropy/utils/misc.py#L366